release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -167,6 +167,26 @@ A new import value, `importMode`, has been added to the `importPolicy` parameter
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-ovn-kubernetes-default-network-plug-in"]
+==== OVN-Kubernetes is now the default networking plug-in
+
+When installing a new cluster the OVN-Kubernetes network plug-in is the default networking plug-in. For all prior versions of {product-title}, OpenShift SDN was the default networking plug-in.
```

Comment:
```suggestion
When installing a new cluster, the OVN-Kubernetes network plugin is the default networking plugin. For all prior versions of {product-title}, OpenShift SDN is the default networking plugin.
```

- In the above suggestion, I assume Openshift SDN is still the default plugin for prior releases which should still be in present tense.

- `plugin` revision - https://redhat-documentation.github.io/supplementary-style-guide/#plugin since this is a new content.

Reply:
@snarayan-redhat, oops, thanks!